Fixes many broken links. (#67)
cloudflare/workers-docs-engine: src/content/reference/runtime-apis/kv.md

This method returns a `Promise` that you should `await` on in order to verify a successful update.

You can also [write key-value pairs from the command line with

> Wrangler](/reference/wrangler-cli/commands#kv:key).

Author: ```suggestion
Wrangler](/reference/wrangler-cli/commands#kvkey).
```

Reviewer: How does this syntax work? The heading looks like this #### `kv:key`

Author: I just navigated to the page and clicked on the headers to see what fragment URL they're using.

Long story short, we're using `remark-slug` to generate these slugs automatically, and it's using "GitHub's algorithm" for generating slugs from headings: https://github.com/remarkjs/remark-slug

Finally, you can [write data via the API](https://api.cloudflare.com/#workers-kv-namespace-write-key-value-pair).

Due to the eventually consistent nature of Workers KV, it's a common pattern to write data via Wrangler or the API, but read the data from within a worker.

#### Writing data in bulk

You can [write more than one key-value pair at a time with

> wrangler](/reference/wrangler-cli/commands#kv:bulk) or [via the

Author: ```suggestion
wrangler](/reference/wrangler-cli/commands#kvbulk) or [via the
```

These assume that `secondsSinceEpoch` and `secondsFromNow` are variables defined elsewhere in your Worker code.

You can also [write with an expiration on the command line via

> Wrangler](/reference/wrangler-cli/commands#kv:key) or [via the

Author: ```suggestion
Wrangler](/reference/wrangler-cli/commands#kvkey) or [via the
```

}
```

You can also [read key-value pairs from the command line with

> wrangler](/reference/wrangler-cli/commands#kv:key).

Author: ```suggestion
wrangler](/reference/wrangler-cli/commands#kvkey).
```

}
```

> You can also [list keys on the command line with Wrangler](/reference/wrangler-cli/commands#kv:key) or [via the API](https://api.cloudflare.com/#workers-kv-namespace-list-a-namespace-s-keys).

Author: ```suggestion
You can also [list keys on the command line with Wrangler](/reference/wrangler-cli/commands#kvkey) or [via the API](https://api.cloudflare.com/#workers-kv-namespace-list-a-namespace-s-keys).
```
